app/models/tax_return.rb

```diff
@@ -189,6 +189,12 @@ def system_change_status(new_status)
     SystemNote::StatusChange.generate!(tax_return: self, old_status: self.status, new_status: new_status)
     self.status = new_status
   end
+
+  def send_client_completion_survey
+    if saved_change_to_status? && %w[file_accepted file_rejected file_mailed].include?(status)
```

Comment:
Move these into a constant called `TERMINAL_STATUSES` in TaxReturnStatus or something like that to indicate these statuses mean that the tax return is done.